src/utils/request.ts

```diff
@@ -10,6 +10,23 @@ export function getQuery (event: CompatibilityEvent) {
 /** @deprecated Use `h3.getQuery` */
 export const useQuery = getQuery
 
+export function getRouterParams (event: CompatibilityEvent): CompatibilityEvent['context'] {
+  // Fallback object needs to be returned in case router is not used (#149)
+  return event.context.params || {}
+}
+
+/** @deprecated Use `h3.getRouterParams` */
+export const useRouterParams = getRouterParams
+
+export function getRouterParam (event: CompatibilityEvent, name: string): CompatibilityEvent['context'][string] {
+  const params = getRouterParams(event)
+
+  return params[name]
+}
+
+/** @deprecated Use `h3.getRouterParam` */
+export const useRouterParam = getRouterParam
```

Comment:
We never introduce `useRouter*` utils. Then don't need to add and deprecate. deprecation of other utils was that they could be used in existing projects.

Reply:
Deleted! 🙋‍♀️ 

819d133